app/models/request.rb

```diff
@@ -0,0 +1,5 @@
+class Request < ActiveRecord::Base
+  validates :content presence: true
+  validates :requester_id, :responder_id, presence: true, numericality: { only_integer: true }
```

Comment:
We are missing associations. 

Associations on this model. 

```
belongs_to :requester, class_name ="User"
belongs_to :responder, class_name="User"
```

With those associations in place, we can do this for validations:

```
validates :requester, :responder, presence: true
```